.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__ . '/src')
+;
+
+$config = new PhpCsFixer\Config();
+return $config->setRules([
+    '@Symfony' => true,
+    'array_syntax' => ['syntax' => 'short'],
```

Comment:
Jak w pozostałych, pytanie o default z uwagi na to co jest tutaj:  https://cs.symfony.com/doc/rules/array_notation/array_syntax.html